gammapy/background/energy_offset_array.py

```diff
@@ -104,3 +127,73 @@ def plot_image(self, ax=None, offset=None, energy=None, **kwargs):
         image = ax.imshow(self.data, extent=extent, **kwargs)
         plt.colorbar(image)
         return ax
+
+    @classmethod
+    def read(cls, filename):
+        """Create `EnergyOffsetArray` from  fits file.
+
+        Parameters
+        ----------
+        filename : str
+            File name
+        """
+        hdu_list = fits.open(filename)
+        return cls.from_fits(hdu_list)
+
+    @classmethod
+    def from_fits(cls, hdu):
+        """Read `EnergyOffsetArray` from a fits binary table.
+
+        Parameters
+        ----------
+        hdu : `~astropy.io.fits.BinTableHDU`
+            HDU binary table for the EnergyOffsetArray.
+
+        Returns
+        -------
+        cube : `~gammapy.background.EnergyOffsetArray`
+            EnergyOffsetArray object.
+        """
+
+        header = hdu[1].header
+        data = hdu[1].data
+        # get offset and energy binning
+        offset_edges = _make_bin_edges_array(data['THETA_LO'].squeeze(), data['THETA_HI'].squeeze())
+        energy_edges = _make_bin_edges_array(data['ENERG_LO'].squeeze(), data['ENERG_HI'].squeeze())
+
+        # get data
+        energy_offset_array = data['EnergyOffsetArray'].squeeze()
+        return cls(energy_edges, offset_edges, energy_offset_array)
+
+    def to_table(self):
+        """Convert `EnergyOffsetArray` to astropy table format.
+
+        Returns
+        -------
+        table : `~astropy.table.Table`
+            Table containing the EnergyOffsetArray.
+        """
+        # table
+        table = Table()
+        table['THETA_LO'] = [self.offset[:-1]]
+        table['THETA_HI'] = [self.offset[1:]]
+        table['ENERG_LO'] = [self.energy[:-1]]
+        table['ENERG_HI'] = [self.energy[1:]]
+        table['EnergyOffsetArray'] = [Quantity(self.data, " u ")]
```

Comment:
Change `EnergyOffsetArray` -> `bkg`

This is the column name according to our current spec:
http://gamma-astro-data-formats.readthedocs.org/en/latest/irfs/background/index.html#bkg-2d-format

Actually, now that I'm thinking about this ... we will want to store COUNTS, LIVETIME and RATE soon, and probably what is called `bkg` in the spec is only the rate.

So if you prefer, you can also leave this as-is and then make a proposal in the other PR where you add the class that represents the 3 arrays in the FITS I/O methods there.
It could be done via a keyword argument passed into this method or via a data member of ...